frontend/src/component/insights/InsightsFilters.tsx

```diff
@@ -34,13 +34,17 @@ export const InsightsFilters: FC<IFeatureToggleFiltersProps> = ({
                 options: [],
                 filterKey: 'from',
                 dateOperators: ['IS'],
+                fromFilterKey: 'from',
+                toFilterKey: 'to',
             },
             {
                 label: 'Date To',
                 icon: 'today',
                 options: [],
                 filterKey: 'to',
                 dateOperators: ['IS'],
+                fromFilterKey: 'from',
```

Comment:
if the from/to filter keys are present then the date selection component automatically gets presets working